python/lsst/pipe/tasks/htmIndexEngine.py

```diff
@@ -133,6 +133,135 @@ def run(self, parsedCmd):
                 result = result,
             )
 
+class IngestIndexedReferenceTask(pipeBase.CmdLineTask):
+    ConfigClass = IngestIndexedReferenceConfig
+    RunnerClass = IngestReferenceRunner
+    _DefaultName = 'IngestIndexedReferenceTask'
+
+    _flags = ['photometric', 'resolved', 'variable']
+
+    @classmethod
+    def _makeArgumentParser(cls):
+        parser = pipeBase.ArgumentParser(name=cls._DefaultName)
```

Comment:
I think you want `InputOnlyArgumentParser`, even though you're writing outputs, so the outputs don't end up in a single output.